test/parallel/test-net-connect-options-port.js

```diff
@@ -76,14 +76,14 @@ const net = require('net');
   const expectedConnections = 72;
   let serverConnected = 0;
 
-  const server = net.createServer(common.mustCall(function(socket) {
+  const server = net.createServer(common.mustCall((socket) => {
     socket.end('ok');
     if (++serverConnected === expectedConnections) {
       server.close();
     }
   }, expectedConnections));
 
-  server.listen(0, 'localhost', common.mustCall(function() {
+  server.listen(0, 'localhost', common.mustCall(() => {
     const port = this.address().port;
```

Comment:
```suggestion
    const port = server.address().port;
```

Explicitly mentioning Request changes so that this commit is not mistakenly landed.